src/eventgrid/azext_eventgrid/_params.py

```diff
@@ -84,6 +85,8 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements
         c.argument('odata_query', arg_type=odata_query_type)
         c.argument('domain_name', arg_type=domain_name_type)
         c.argument('domain_topic_name', arg_type=domain_topic_name_type)
+        c.argument('allow_traffic_from_all_ips', arg_type=get_three_state_flag(), options_list=['--allow-traffic-from-all-ips'], help="Allow traffic from all IPs.")
```

Comment:
The `options_list` here can be omitted since `allow_traffic_from_all_ips ` will be mapped to `allow-traffic-from-all-ips` by default.